recipe/build.sh

```diff
@@ -1,36 +1,37 @@
 #!/bin/bash
 
-mkdir build
+mkdir build || true
 cd build
 
-
 declare -a CMAKE_PLATFORM_FLAGS
 if [[ ${target_platform} =~ osx.* ]]; then
     # only needed until conda-forge toolchain 2.3.0 is used
     #   https://github.com/conda-forge/toolchain-feedstock/pull/47
     CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
-else
+elif [[ ${target_platform} =~ linux.* ]]; then
     CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
     # link transitive ADIOS1 libraries during build of intermediate wrapper lib
-    export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,$PREFIX/lib"
+    export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
 fi
 
+bash ${SRC_DIR}/.travis/download_samples.sh
```

Comment:
needs to be sourced as:
```suggestion
source ${SRC_DIR}/.travis/download_samples.sh
```
nevertheless, the build should continue gracefully without samples: https://github.com/openPMD/openPMD-api/blob/4cbe73c1f824b6d8509131fffe6adc20f7670cdc/CMakeLists.txt#L834

Reply:
The tests area still run without test data though. That is clear.